src/Features/header/components/CityLanguageSelect/City/CitySelect.tsx

```diff
@@ -0,0 +1,81 @@
+import Link from 'next/link';
+import React, { ReactElement, useState, useEffect } from 'react';
+import { setCookie, getCookie, CookieValueTypes } from 'cookies-next';
+import { Modal } from '../../../../../components/Modal/Modal';
+
+interface CitySelectProps {
+  myValue: string | undefined;
+}
+
+export const CitySelect: React.FC<CitySelectProps> = ({ myValue }): ReactElement => {
```

Comment:
It is better to omit `FC` because it is old https://www.reddit.com/r/reactjs/comments/ys70t9/is_is_still_problematic_to_use_reactfc_if_our/
https://www.reddit.com/r/reactjs/comments/vx5qpa/is_fc_still_discouraged/